features/guidance/destination-signs.feature

```diff
@@ -0,0 +1,35 @@
+@routing @guidance
+Feature: Destination Signs
+
+    Background:
+        Given the profile "car"
+
+    Scenario: Car - route name assembly with destination signs
+        Given the node map
+          | a | b |
+          | c | d |
+          | e | f |
+          | g | h |
+          | i | j |
+          | k | l |
+          | m | n |
+
+        And the ways
+          | nodes | name | ref | destination    | destination:ref |
+          | ab    | AB   | E1  |                |                 |
+          | cd    | CD   |     | Berlin         |                 |
+          | ef    | EF   |     | Berlin         | A1              |
+          | gh    |      |     | Berlin         | A1              |
+          | ij    |      |     | Berlin         |                 |
+          | kl    | KL   | E1  | Berlin         | A1              |
+          | mn    | MN   |     | Berlin;Hamburg | A1;A2           |
+
+        When I route I should get
+          | from | to | route                                                     |
+          | a    | b  | AB (E1),AB (E1)                                           |
+          | c    | d  | CD (Berlin),CD (Berlin)                                   |
+          | e    | f  | EF (A1: Berlin),EF (A1: Berlin)                           |
+          | g    | h  | ,                                                         |
+          | i    | j  | ,                                                         |
```

Comment:
I thought this was cucumber assembling the responses? I got this from https://github.com/Project-OSRM/osrm-backend/blob/e88106e990336e6b38dfadbdd85f147c6f20a0f8/features/guidance/ramp.feature#L228-L229 as I could not find documentation on it.